### ISSUE TITLE
Add an option to use the domain from the Sender: header instead of From: for DKIM

### ISSUE THREAD

**Issue by ppadron** [zone-eu/zone-mta#159]
Hello!

I am still diving into ZoneMTA so I'm not sure what would be the best way to achieve this:

When the From: domain is not properly setup for a DKIM signature, you can add a Sender: header with a default domain (with valid DKIM records) and put the From: formatted as VERP in the localpart.

Example:

```
From: pedro@example.com (example.com is not properly setup)
Sender: pedro=example.com@mymailingsoftware.com (mymailingsoftware.com is properly setup)
```

Some MUAs like Gmail will display the sender as pedro@example.com via mymailingsoftware.com.

For this to work, we would need to set a default Sender: domain to use its DKIM signature instead of the From: domain.

Should I just create my own plugin using the dkim.js as reference or do you think this is useful enough to be an option in the existing plugin?

**Comment by andris9** [comment]
Yeah, I guess buulding your own plugin would be the easiest way to go. You can get the sender header value like this

    let sender=delivery.headers.getFirst('sender');
